docs/pages/guides/customizing-metro.md

```diff
@@ -162,7 +154,15 @@ module.exports = config;
 
 Here are all of the [default Uglify options](https://github.com/facebook/metro/blob/b629f44239bbb3414491755185cf19b5834b4b7a/packages/metro-config/src/defaults/index.js#L94-L111) applied in Metro bundler.
 
-### Using Terser
+### Minification
```

Comment:
```suggestion
## Minification
```

Let's make Minification a level 2 header so the next headers related to alternative solutions are nested within this one.